lib/discourse_login_client_authenticator.rb

```diff
@@ -18,7 +18,7 @@ class DiscourseLoginClientStrategy < ::OmniAuth::Strategies::OAuth2
         username: access_token.params["info"]["username"],
         name: access_token.params["info"]["username"],
```

Comment:
Unrelated to your PR, but we should look into this line... I'm not sure we need to set `name = username`.